go/pkg/idl/testdata/example.stef

```diff
@@ -0,0 +1,38 @@
+// Records of events that happened with books. This is the main record struct.
+struct BookRecords root {
```

Comment:
To clarify, is the `root` keyword just for readability, given that root fields can be deducted?

Reply:
Not just for readability. In theory you can have a root struct (which means we generate Reader/Writer), which can still be a child struct in another struct. It is pretty unlikely, but not prohibited. So `root` keyword simply means: generate Reader/Writer.